README.md

```diff
@@ -99,6 +88,23 @@ This will start the Play application, which usually listens on port `9111`. Maki
 
 To make the site reachable as `contributions.thegulocal.com` (necessary for register/sign-in functionality) you then need to make sure NGINX is configured and running as described in [`/nginx/README.md`](./nginx/README.md).
 
+
+### Running the play app manuall
+```
+sbt devrun
+```
+
+### Watch files for changes
+
+```
+npm run watch
+```
+
+This runs [webpack-dev-server](https://webpack.github.io/docs/webpack-dev-server.html) on port 7777, proxying requests to 9111. In this case, the bundled JS is stored in memory, which allows for much faster recompilation on changes, but the main.js file won't be created. You can also access localhost:7777/webpack-dev-server/\<route\> for a live reload environment. 
```

Comment:
yeah I think there's a broader conversation to have on how to start the dev server now we have webpack.
Could that be integrated with sbt? or should we have a local architecture with Ngninx -> webpack -> play ?